.github/workflows/infrastructure.yml

```diff
@@ -38,36 +36,36 @@ jobs:
           # Admin needs to be enabled for App Service continuous deployment
           az acr create --name tfsdev --sku Standard --admin-enabled true
 
-  test-rg:
-    name: Test resource group
+  staging-rg:
+    name: 2. 'Staging' resource group
     runs-on: [self-hosted, linux]
     steps:
     - uses: azure/login@v1
       with:
-        creds: ${{ secrets.AZURE_DIGITAL_TEST }}
+        creds: ${{ secrets.AZURE_DIGITAL_STAGING }}
 
     - name: Defaults
       uses: azure/cli@v1.0.0
       with:
         inlineScript: |
           az configure --defaults location=uksouth
-          az configure --defaults group=${{ secrets.TEST_RESOURCE_GROUP }}
+          az configure --defaults group=${{ secrets.STAGING_RESOURCE_GROUP }}
```

Comment:
do we have the credentials for Staging resource group?

Reply:
Yep
![image](https://user-images.githubusercontent.com/32125108/156922480-e10d2711-b8b2-4d7c-9ee7-1c8d3146ffd2.png)